### PR TITLE
Add missing repository integrity docs for Health API

### PR DESCRIPTION
**Pull request by nielsbauman**
Follow up on https://github.com/elastic/elasticsearch/pull/104614 where I forgot to add the new details to the docs.

### CODE REVIEW

File: docs/reference/health/health.asciidoc [hunk 1]
```diff
@@ -356,6 +356,14 @@ watermark threshold>>.
     (Optional, array of strings) If corrupted repositories have been detected in the system, the names of up to ten of
     them are displayed in this field. If no corrupted repositories are found, this detail is omitted.
 
+`unknown_repositories`::
+    (Optional, int) The number of repositories on the system that have been determined to be unknown. If there are
```
Comment (gmarouli) [inline docs/reference/health/health.asciidoc:360]: Maybe here we should stress the fact that this is a node specific issue. For example, 

> The number of repositories that have been determined to be unknown by at least one node in the system.

What do you think?

File: docs/reference/health/health.asciidoc [hunk 2]
```diff
@@ -356,6 +356,14 @@ watermark threshold>>.
...
+    no corrupted repositories detected, this detail is omitted.
+
+`invalid_repositories`::
+    (Optional, int) The number of repositories on the system that have been determined to be invalid. If there are
```
Comment (gmarouli) [inline docs/reference/health/health.asciidoc:364]: Same here.

**Review by gmarouli — APPROVED** [review]
LGTM, I added a small suggestion to rephrase the details a bit.

File: docs/reference/health/health.asciidoc (hunk 1 above)
Comment (nielsbauman) [reply docs/reference/health/health.asciidoc:360]: Good point, thanks for the suggestion :)

**Pull request merged**